package.json

```diff
@@ -41,6 +41,8 @@
     "debug": "^3.0.0",
     "depd": "^1.1.0",
     "dottie": "^2.0.0",
+    "env-cmd": "^5.0.0",
+    "flat": "^4.0.0",
```

Comment:
I dont want to include other dependency which will be used in one function only, may be we can use lodash to get leaf keys ?